frontend/src/components/PlannerChart.tsx

```diff
@@ -82,7 +82,11 @@ export const PlannerChart: React.FC<{
         ...tasks.map((task) => {
           const work =
             calcTaskAverageRating(TaskRatingDimension.RequiredWork, task) || 0;
-          const value = calcTaskValueSum(task);
+          const valueRatings = totalRatingsByDimension(task).get(
+            TaskRatingDimension.BusinessValue,
+          ) ?? { sum: 0, count: 1 };
+          const value = valueRatings.sum ?? 0;
+          // totalRatingsByDimension(task).get(TaskRatingDimension.BusinessValue).sum ?? 0
```

Comment:
Is this still in progress, or just left over?

Reply:
Forgot to delete that comment, everything is ready